packages/core/src/generateAppDeclarationFile.test.ts

```diff
@@ -0,0 +1,54 @@
+import ts from "typescript"
+import generateAppDeclarationFile from "./generateAppDeclarationFile"
+
+import parseSamenApp from "./parseSamenApp"
+import { createTestProgram } from "./tsTestUtils"
+
+function generate(prog: ts.Program): string {
+  // if (prog.getSemanticDiagnostics().length) {
```

Comment:
comment

Reply:
These are some test logs I use from time to time.